src/service-bus-dotnet-management/AppOptions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.Azure.Management.ServiceBus.Models;
+
+namespace service_bus_dotnet_management
```

Comment:
This was the original namespace for the sample. I didn't touch it.
Will rename namespace to `AzureServiceBusDotNetManagement`